app/controllers/users_controller.rb

```diff
@@ -0,0 +1,38 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+    redirect_to user_path(current_user) if logged_in?
+  end
+
+  def create
+    @user = User.create(signup_params)
+    if @user.save
+      flash[:success] = 'Account is sucessfully created'
+      session[:user_id] = @user.id
+      session[:username] = @user.name
+      redirect_to user_path(@user)
+    else
+      flash[:danger] = 'You are an existing user...  Please sign in'
+      render 'new'
+    end
+  end
+
+  def show
```

Comment:
- [ ] Also lists the events created by user, i.e. `@created_events =  current_user.events` here instead of having `@user.events` directly at the view.

Reply:
Fixed